docs/pages/management/export-audit-events/elastic-stack.mdx

```diff
@@ -43,93 +43,15 @@ will use for authentication.
 Follow the instructions for your environment to install the Event Handler
 plugin on your Logstash host:
 
-<Tabs dropdownCaption="Method" dropdownSelected="Executable">
-  <TabItem options="Executable" label="Latest Release">
-
-  On the host where you are running Logstash, execute the following commands:
-
-  ```code
-  $ curl -L -O https://get.gravitational.com/teleport-event-handler-v(=teleport.plugin.version=)-linux-amd64-bin.tar.gz
-  $ tar -zxvf teleport-event-handler-v(=teleport.plugin.version=)-linux-amd64-bin.tar.gz
-  ```
-
-  Move the `teleport-event-handler` binary into your `PATH`.
-  </TabItem>
-
-  <TabItem options="Docker" label="Latest Release ">
-
-  On the host where you are running Logstash, execute the following command:
-
-  ```code
-  $ docker pull public.ecr.aws/gravitational/teleport-plugin-event-handler:(=teleport.plugin.version=)
-  ```
-
-  </TabItem>
-
-  <TabItem options="Docker" label="From Source">
-  You must have Docker Desktop installed on your Logstash host. 
-
-  Log in to the host and execute the following commands to build the plugin:
-
-  ```code
-  $ git clone https://github.com/gravitational/teleport-plugins.git --depth 1
-  $ cd teleport-plugins/event-handler/build.assets
-  $ make build
-  ```
-
-  You can find the compiled binary within your clone of the `teleport-plugins`
-  repo, with the file path, `event-handler/build/teleport-event-handler`. Move
-  this into your `PATH`.
-
-  </TabItem>
-  <TabItem options="Executable" label="From Source ">
-
-  You will need Go >= (=teleport.golang=) installed.
-
-  Run the following commands on your Logstash host:
-
-  ```code 
-  $ git clone https://github.com/gravitational/teleport-plugins.git --depth 1
-  $ cd teleport-plugins/event-handler
-  $ go build
-  ```
-
-  The resulting executable will have the name `event-handler`. To follow the
-  rest of this guide, rename this file to `teleport-event-handler` and move it
-  into your `PATH`.
-
-  </TabItem>
-</Tabs>
+(!docs/pages/includes/install-event-handler.mdx!)
 
 ### Generate a starter config file
 
 Generate a configuration file with placeholder values for the Teleport Event
 Handler plugin. Later in this guide, we will edit the configuration file for
 your environment.
 
-On the host where you are running Logstash and the Event Handler plugin, execute
-the following command to generate a sample configuration that we will edit
-later.  Replace `mytenant.teleport.sh` with the DNS name of your Teleport Proxy
-Service or Teleport Cloud tenant:
-
-```code
-$ teleport-event-handler configure . mytenant.teleport.sh
-```
-
-The `teleport-event-handler configure` command generates several files that you
-will refer to when generating credentials for the plugin and configuring a
-Logstash pipeline:
-
-|File(s)|Purpose|
-|---|---|
-| `ca.crt` | Self-signed CA certificate that the Event Handler plugin and Logstash will use to validate one another's TLS certificates.|
```

Comment:
This table includes some Logstash-specific context that is lost in the partial

Reply:
Resolved in fad1a0a6a29ee90e6baaff39b4dae4a6767c6ed9